styles/globals.css

```diff
@@ -614,10 +614,30 @@ ul, ol {
     display: block !important; }
 } 
 
+#accordion details > summary {
+  list-style-type: none;
+  outline: none;
+  cursor: pointer;
+  font-size: 20px;
+}
+#accordion details > summary:hover {
+  background-color: rgba(207, 207, 207, 1)
+}
+#accordion details > summary:active, #accordion details > summary:focus {
+  border: 1px solid black;
+}

```

Comment:
The focus and hover states should be a slightly lighter shade of gray, and active state should be a darker gray (like what the hover is now)

Using the report a problem component as an example, the hover and focus background color is #CFCFCF and the active background is #BBBFC5.